apps/ledger-live-desktop/static/i18n/en/app.json

```diff
@@ -4958,6 +4958,10 @@
     }
   },
   "errors": {
+    "ReplacementTransactionUnderpriced": {
+      "title": "Replacement transaction underpriced",
+      "description": "The network fee must be at least 10% higher than the fee of the original transaction."
```

Comment:
Same question

Reply:
Same thing as above https://github.com/LedgerHQ/ledger-live/pull/4886#discussion_r1368380567

And event slightly more complicated, since this 10% value is actually not specific to us but rather a minimum defined by most node implementations.

cf. https://github.com/LedgerHQ/ledger-live/blob/51473ff1c859f71765667acd8aa5c8d6ce24cf76/libs/coin-evm/src/getTransactionStatus.ts#L298-L305

This 10% is a hard limit, but it does not mean it will necessarily work with this value (that's why is states "at least"), depending on network activity at the time and other parameters (including, but not limited to, [the values defined by LL](https://github.com/LedgerHQ/ledger-live/blob/f521f5ba0a7a5c85d32ae7f0025ddae2e0d75fe2/libs/env/src/env.ts#L743-L757))

Here, we have multiple options:
1. not change the message, which is not factually false (to me "at least" means that it's not guaranteed to work if I update by the proposed amount, but I might be wrong here)
2. if 1 is not clear enough (which I can agree on), it could be rephrased. The basic intent of the message is that the proposed fees are to low and must be updated (by 10% minimum but maybe more). Any proposal is welcomed.
3. get rid of the percent value if misleading or unclear, and go with a simpler message like "The network fees are too low and must be increased"